mysql-test/suite/rocksdb/include/bulk_load_unsorted.inc

```diff
@@ -1,14 +1,11 @@
---disable_warnings
-DROP TABLE IF EXISTS t1;
---enable_warnings
-
```

Comment:
Missing `--source include/count_sessions.inc` here

Reply:
Now how the heck did I do that, this is the file that started me in on this work :-\